src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1385,7 +1385,7 @@ private function addCacheSection(ArrayNodeDefinition $rootNode, callable $willBe
                             ->info('System related cache pools configuration.')
                             ->defaultValue('cache.adapter.system')
                         ->end()
-                        ->scalarNode('directory')->defaultValue('%kernel.share_dir%/pools/app')->end()
+                        ->scalarNode('directory')->defaultValue('%kernel.share_dir%/cache/%kernel.environment%/pools/app')->end()
```

Comment:
let's reduce the nesting?
```suggestion
                        ->scalarNode('directory')->defaultValue('%kernel.share_dir%/%kernel.environment%/app_cache')->end()
```

Reply:
it would be then kind of mix between environment specific dirs and none environment dirs:

```
 - var/share
    - dev
       - app_cache
       - http_cache
    - indexes (none env dir)
    - prod
       - app_cache
       - http_cache
    - stage
       - app_cache
       - http_cache
    - storage (none env dir)
```

Reply:
hum right, then let's have the env after the folder

Reply:
Okay looks like this now:

```
 - var/share
    - app_cache
       - dev
       - stage
       - prod
    - http_cache
       - dev
       - stage
       - prod
    - indexes (none env dir)
    - storage (none env dir)
```